operations/research-and-development/product/product-planning/how-to-use-productboard.md

```diff
@@ -113,113 +114,113 @@ Two very important notes regarding this integration:
 
-### 2b - Adding feedback
+### 2b Adding feedback
 
-New feedback from users and customers is sent to the [Insights Board](https://mattermost.productboard.com/insights). Typically these are submitted by Product Managers, Support, Customer Success Managers, or Sales through integrations with email, Zapier, Zendesk, or others.
+New feedback from users and customers is sent to the [Insights board](https://mattermost.productboard.com/insights). Typically these are submitted by Product Managers, Support, Customer Success Managers, or Sales through integrations with email, Zapier, Zendesk, or others.
 
-### 2c - Creating notes
+### 2c Creating notes
 
-You can add a new note to capture feedback from a customer call. This is used to help identify customer problems, wants/needs, feature shortcomings, and any other areas for improvement that stem from interacting with customers. Essentially, think of this as a place to communicate the most important takeaways from customer calls that the Product team can help you with. 
+You can add a new note to capture feedback from a customer call. This is used to help identify customer problems, wants/needs, feature shortcomings, and any other areas for improvement that stem from interacting with customers. Essentially, think of this as a place to communicate the most important takeaways from customer calls that the Product team can help you with.
 
 Important points about notes:
 
 * Notes are specific to a customer/prospect.
-* Notes can include multiple requests (PMs can apply multiple elements from the note to different features if necessary).
-* If you are not sure if the request or feedback should apply to an existing feature, you can simply capture the feedback as a note, assign it to the PM overseeing that area, or leave it unassigned.  
+* Notes can include multiple requests (Product Managerss can apply multiple elements from the note to different features if necessary).
+* If you are not sure if the request or feedback should apply to an existing feature, you can simply capture the feedback as a note, assign it to the Product Manager overseeing that area, or leave it unassigned.
 
-### 2d - Adding insights directly to an existing feature
+### 2d Adding insights directly to an existing feature
 
-As a **Contributor** in ProductBoard, you have access to view the [master feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list). This is the view where the Product team organizes and prioritizes new features and functionality that is being considered or planned for Mattermost. 
+As a **Contributor** in productboard, you have access to view the [Master Feature list](https://mattermost.productboard.com/feature-board/1097524-master-feature-list). This is the view where the Product team organizes and prioritizes new features and functionality that is being considered or planned for Mattermost.
```

Comment:
Is it possible to view what status one has? I can't find where my role is listed and, are there multiple roles (e.g. Contributor, Viewer etc)?

Reply:
That is a good question. I do not believe there is.

Maybe we can describe the logic we employ to grant contributor/maker access? 
- Makers = PMs and PM leadership
- Contributors: CE/CS/UX/Dev Leadership
- Viewers: Everyone else with `@mattermost.com` email address